djangocms_navigation/templates/admin/djangocms_navigation/menuitem/change_list.html

```diff
@@ -1,6 +1,11 @@
 {# Used for MP and NS trees #}
 {% extends "admin/tree_change_list.html" %}
```

Comment:
This is where we can add the scripts that are currently a template tag "treebeard_js"